src/TodoForm.jsx

```diff
@@ -1,22 +1,29 @@
-import { useRef } from 'react';
+import { useState } from 'react';
```

Comment:
`useRef` is missing. `ref` needs to be integrated with this component to focus on input after submission.